jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py

```diff
@@ -83,26 +81,3 @@ def vue_save_as_fits(self, *args):
         # Let the user know where we saved the file.
         self.hub.broadcast(SnackbarMessage(
             f"Moment map saved to {os.path.abspath(self.filename)}", sender=self, color="success"))
-
-    def _link_moment_data(self):
-        new_len = len(self.app.data_collection)
-
-        # Can't link if there's no world_component_ids
-        pc_new = self.app.data_collection[-1].pixel_component_ids
-
-        # Link to the first dataset with compatible coordinates
-        for i in range(new_len - 1):
-            pc_old = self.app.data_collection[i].pixel_component_ids
-            # If data_collection[i] is also from the moment map
-            if self.app.data_collection[i].meta.get("Plugin", None) == self.__class__.__name__:
-                links = [LinkSame(pc_old[0], pc_new[0]),
-                         LinkSame(pc_old[1], pc_new[1])]
-
-            # Link moment map to cube (pc_old)
-            else:
-                links = [LinkSame(pc_old[1], pc_new[0]),
-                         LinkSame(pc_old[2], pc_new[1])]
```

Comment:
For what it's worth, changing this to `LinkSame(pc_old[0], pc_new[1])]` would have fixed it in the old setup before you deleted this code.